api/envoy/config/core/v3/substitution_format_string.proto

```diff
@@ -65,4 +65,14 @@ message SubstitutionFormatString {
   //   empty string, so that empty values are omitted entirely.
   // * for ``json_format`` the keys with null values are omitted in the output structure.
   bool omit_empty_values = 3;
+
+  // Specify a content_type for text_format. This will be ignored for json_format.
```

Comment:
Why ignore for JSON? I feel there are JSON content type variants you might want to support, e.g. https://stackoverflow.com/questions/477816/what-is-the-correct-json-content-type